packages/playback-core/src/types.ts

```diff
@@ -131,20 +133,21 @@ export type MuxMediaPropTypes = {
   errorTranslator: Options['errorTranslator'];
   disableCookies: Options['disableCookies'];
   playbackId: string;
-  playerInitTime: Options['data']['player_init_time'];
+  playerInitTime: MetaData['player_init_time'];
   preferPlayback: ValueOf<PlaybackTypes> | undefined;
   type: MediaTypes;
   streamType: ValueOf<StreamTypes>;
   startTime: Hls['config']['startPosition'];
   autoPlay?: Autoplay;
   autoplay?: Autoplay;
   preferCmcd: ValueOf<CmcdTypes> | undefined;
+  error?: HTMLMediaElement['error'] | MediaError;
```

Comment:
this one is technically a lie since we should _**always**_ yield our custom `MediaError` (or `null`) in `MuxVideoElement`. However, overriding accessor with completely different type signatures in subclasses gets messy, so went with an expanding type instead.